opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -189,6 +190,53 @@ namespace Opm
         std::vector<int> current_controls_;
     };
 
+    static inline data::Wells report( const WellStateFullyImplicitBlackoil& state,
```

Comment:
I suppose inline is sufficient (pretty sure header-inline implies static behaviour anyway).